astro/src/content/quickstarts/quickstart-javascript-vue-web.mdx

```diff
@@ -0,0 +1,188 @@
+
+```shell
+npm install @fusionauth/vue-sdk
+```
+
+Next, you'll need to configure and active the FusionAuth Vue SDK. You can do this by updating the `src/main.ts` file:
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-vue-web/main/complete-application/src/main.ts"
+            lang="typescript"/>
+
+Our example application is going to have a home page and an account page. The account page will be protected and only visible to logged-in users.
+
+#### Create A Home Page
+
+The next step is to get a basic home page up and running. We’ll take this opportunity to copy in all the images and CSS style files that you’ll need for the application.
+
+Run the following copy command to copy these files from the quickstart repo into your project. This assumes that you checked the quickstart repo out into the parent directory. If that’s not the case, replace the .. below with your actual repo location.
+
+```shell
+cp -r ../complete-application/src/assets src
+```
+
+The home page will be a simple page with a welcome message and a login link. Update the existing `src/views/HomeView.vue`:
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-vue-web/main/complete-application/src/views/HomeView.vue"
+            lang="vue"/>
+
+#### Create An Account Page
+
+The account page will be a simple page that displays a random balance for the logged-in user. Create a new file `src/views/AccountView.vue`:
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-vue-web/main/complete-application/src/views/AccountView.vue"
+            lang="vue"/>
+
+## Authentication
+
+You now have created a basic Vue.js application with a home page and an account page.
+
+Depending on the user's authentication state, the login or logout button should be displayed in the header. For this create a new file `src/components/LogoHeader.vue`:
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-vue-web/main/complete-application/src/components/LogoHeader.vue"
+            lang="vue"/>
+
+Additionally, we want to display different menu items. For this create a new file `src/components/MenuBar.vue`:
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-vue-web/main/complete-application/src/components/MenuBar.vue"
+            lang="jsx"/>
+
+The next step is to tie it all together. Update the `src/App.vue` file to add the router view and header:
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-vue-web/main/complete-application/src/App.vue"
+            lang="vue"/>
+
+And finally we register the routes in `src/router/index.ts`:
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-vue-web/main/complete-application/src/router/index.ts"
```

Comment:
Dan published the library so we can change the `import` path here :tada:

Reply:
got updated to @fusionauth/vue-sdk

https://github.com/sonderformat-llc/fusionauth-quickstart-javascript-vue-web/blob/main/complete-application/src/router/index.ts